lib/applicants/schema.graphql

```diff
@@ -75,3 +76,19 @@ input UpdateApplicantInput {
 type UpdateApplicantResult {
   ok: Boolean!
 }
+
+input ApplicantsFilter {
+  order: [[String]]
```

Comment:
Just want to double check, is there a way to specify that the inner array has 2 elements of type `String` and `SortOrder`?

Reply:
i tried to figure this out and it was pretty tough for an array. We could think about changing the format of a SortOptions to be an array of objects `[{field: string, order: SortOrder}]` instead of `[[string, sortOrder]]` and that would be easier to implement in gql

it looks like its a known issue https://github.com/graphql/graphql-spec/issues/423

Reply:
Okay good idea, will add this investigation to backlog.